### PR TITLE
New fields and script additions

### PR DESCRIPTION
**Pull request by dkimdkimdkim**

### CODE REVIEW

File: scripts/es.sh [hunk 1]
```diff
@@ -29,8 +29,10 @@ function es {
     if $PRETTY_PRINT; then
         local py_cmd="import json; print json.dumps(json.loads('$res'), indent=2)"
         if ! $COLORS || [ "`which pygmentize`" == "" ]; then
+            echo "NO PYGMENTIZE"
             python -c "$py_cmd"
         else
+            echo "YES PYGMENTIZE"
```
Comment (NatalieWolfe) [inline scripts/es.sh]: Remove these debug logging bits.

File: scripts/test_data.sh [hunk 2]
```diff
@@ -6,19 +6,71 @@ function insert_data {
...
+    # Discern from filename of form join_<entity1>_<entity2>*.json that entity1 is parent and entity2 is child.
+    # e.g. (join_artist_releases.json -> parent:artist, child:release)
+    parent_type="$( cut -d '_' -f 1 <<< "$1" )"
+    child_type_plural="$( cut -d '_' -f 2 <<< "$1" )"
+    child_type="${child_type_plural%?}"
+
+    current_line_type="parent"
+
+    while read line; do
+        if [[ -z $line ]]; then
```
Comment (NatalieWolfe) [inline scripts/test_data.sh]: Do not need double brackets on this statement.

File: scripts/es.sh (hunk 1 above)
Comment (dkimdkimdkim) [inline scripts/es.sh]: Oops yeah, done

On Fri, Apr 10, 2015 at 10:55 AM, Natalie Wolfe <notifications@github.com>
wrote:

> In scripts/es.sh
> <https://github.com/pulselocker/elasticsearch/pull/26#discussion_r28166021>
> :
>
> >              python -c "$py_cmd"
> >          else
> > +            echo "YES PYGMENTIZE"
>
> Remove these debug logging bits.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/pulselocker/elasticsearch/pull/26/files#r28166021>.
>



-- 







*/////////////////////////////////////////david kimsoftware
engineere: david@pulselocker.com <david@pulselocker.com>t: +1.415.729.3970
x719skype: dkimdkimdkim  web: pulselocker.com
<http://pulselocker.com/>twitter: @pulselocker
<http://twitter.com/pulselocker>facebook: facebook.com/pulselocker
<http://facebook.com/pulselocker>*

-- 

*The information in this email and subsequent attachments may 
contain confidential information that is intended solely for the attention 
and use of the named addressee(s). This message or any part thereof must 
not be disclosed, copied, distributed or retained by any person 
without authorization without expressed written permission from the sender.*

File: scripts/test_data.sh (hunk 2 above)
Comment (dkimdkimdkim) [inline scripts/test_data.sh]: done

On Fri, Apr 10, 2015 at 10:56 AM, Natalie Wolfe <notifications@github.com>
wrote:

> In scripts/test_data.sh
> <https://github.com/pulselocker/elasticsearch/pull/26#discussion_r28166130>
> :
>
> > +# child
> > +#
> > +# parent
> > +#
> > +# parent
> > +function insert_join_from_file {
> > +    # Discern from filename of form join_<entity1>_<entity2>*.json that entity1 is parent and entity2 is child.
> > +    # e.g. (join_artist_releases.json -> parent:artist, child:release)
> > +    parent_type="$( cut -d '_' -f 1 <<< "$1" )"
> > +    child_type_plural="$( cut -d '_' -f 2 <<< "$1" )"
> > +    child_type="${child_type_plural%?}"
> > +
> > +    current_line_type="parent"
> > +
> > +    while read line; do
> > +        if [[ -z $line ]]; then
>
> Do not need double brackets on this statement.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/pulselocker/elasticsearch/pull/26/files#r28166130>.
>



-- 







*/////////////////////////////////////////david kimsoftware
engineere: david@pulselocker.com <david@pulselocker.com>t: +1.415.729.3970
x719skype: dkimdkimdkim  web: pulselocker.com
<http://pulselocker.com/>twitter: @pulselocker
<http://twitter.com/pulselocker>facebook: facebook.com/pulselocker
<http://facebook.com/pulselocker>*

-- 

*The information in this email and subsequent attachments may 
contain confidential information that is intended solely for the attention 
and use of the named addressee(s). This message or any part thereof must 
not be disclosed, copied, distributed or retained by any person 
without authorization without expressed written permission from the sender.*

**Pull request merged**